config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only: [:new, :create]
```

Comment:
Resources implies that we will manage `:id` in our routes, but new/create do not need id's?  Could we get away with just `resource`?